### ISSUE TITLE
[BE] Transferring email content to backend (TalktixChatAPI)

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#39]
Extension to #38 #40

- send e-mail body and respective subject to TalktixChatAPI so a ticket can be created

## Acceptance criteria
- [ ] e-mail content is sent in the correct format to TalktixChatAPI
- [ ] AI receives whole content correctly

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
